testing/util/lightclient.go

```diff
@@ -94,9 +103,10 @@ func WithFinalizedCheckpointInPrevFork() LightClientOption {
 }
 
 // WithSupermajority specifies whether the sync committee bits have supermajority or not
-func WithSupermajority() LightClientOption {
+func WithSupermajority(increaseNumParticipantsBy uint64) LightClientOption {
 	return func(l *TestLightClient) {
 		l.supermajority = true
+		l.increaseActiveParticipantsBy = increaseNumParticipantsBy
```

Comment:
why did we want to have this as the same functional option instead of a new one?